tap_github/streams.py

```diff
@@ -22,16 +22,27 @@ def get_url_params(
         """Return a dictionary of values to be used in URL parameterization."""
         assert context is not None, f"Context cannot be empty for '{self.name}' stream."
         params = super().get_url_params(context, next_page_token)
-        params["q"] = context["search_query"]
+        if "search_query" in context:
+            # we're in search mode
+            params["q"] = context["search_query"]
+        else:
+            # we are in repository list mode
+            params["q"] = f"repo:{context['org']}/{context['repo']}"
```

Comment:
For this use case, I think there's a more performant `repositories` endpoint to use, rather than the `search` endpoint.

Reply:
In line 59, where we have `path = "/search/repositories"`, I think something like this might do the trick:

```py
    @property
    def path(cls) -> str:
        if "searches" in self.config:
            return "/search/repositories"
        else:
            # the `repo` and `org` args will be parsed from the partition's `context`
            return "/repos/{org}/{repo}"         
```

Reply:
> According to the docs, the ?q=repo:org/repo should return exactly the repo you specify (I tried searching for one that does not exist, the call throws an error). A bit of testing seems to confirm that.

Awesome. Given this ☝️, I'm actually totally fine with either approach. While the `repos` endpoint seems more purpose built for one repo at a time, there are pros and cons of each approach which we could always work through in a future update.